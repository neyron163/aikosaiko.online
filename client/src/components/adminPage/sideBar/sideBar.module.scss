:local {
    .container {
        width: 30%;
    }
}
