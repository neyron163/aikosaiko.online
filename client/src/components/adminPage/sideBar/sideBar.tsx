import React from 'react';

import {LinkButton} from 'components/linkButton/linkButton';

import s from './sideBar.module.scss';

/**
 * Side bar on the left
 */
export const SideBar = () => (
    <div className={s.container}>
        <LinkButton path="/admin/createProducts">Create</LinkButton>
        <LinkButton path="/admin/listProducts">Products</LinkButton>
    </div>
);
