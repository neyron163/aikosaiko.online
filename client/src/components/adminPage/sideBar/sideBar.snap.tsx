import React from 'react';
import renderer from 'react-test-renderer';

import {SideBar} from './sideBar';

jest.mock('components/linkButton/linkButton', () => ({LinkButton: 'LinkButton'}));

describe('<SideBar />', () => {
    it('should render correctly', () => {
        const tree = renderer.create(<SideBar />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
