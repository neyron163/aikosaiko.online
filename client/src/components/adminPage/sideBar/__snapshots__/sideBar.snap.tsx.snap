// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SideBar /> should render correctly 1`] = `
<div
  className="container"
>
  <LinkButton
    path="/admin/products"
  >
    Products
  </LinkButton>
</div>
`;
